cmd/virt-freezer/main.go

```diff
@@ -42,6 +44,17 @@ func getGrpcClient() (cmdclient.LauncherClient, error) {
 	return client, err
 }
 
+// hasPausedCondition checks if the VM to be freezed/unfreezed is paused
+func hasPausedCondition(client cmdclient.LauncherClient) bool {
+	domain, exists, err := client.GetDomain()
+	if err != nil {
+		log.Log.Reason(err).Error("Failed to get domain")
+		os.Exit(1)
+	}
+
+	return exists && domain.Status.Status == api.Paused
```

Comment:
rather than checking paused, maybe it would be better to only freeze/unfreeze when status is `Running`?

Reply:
I'm not too well-versed on what'd happen if we freeze in the other statuses, but it makes sense. I'll change it :+1: